### PR TITLE
Various dev setup updates

### PR DESCRIPTION
**Pull request by Fryguy**
Reviewing these steps with some new developers, I found some things lacking or unclear, so this PR fixes those.

@chessbyte Please review.  (easier to review with whitespace diffs off)

### CODE REVIEW

File: developer_setup.md
Comment (NickLaMuro) [inline developer_setup.md:77]: I wonder if it makes sense to instead suggest installing `postgresql@10` `brew`, and equivalent for `dnf`/`yum`/`apt`.  Not sure if it is worth doing in this PR, but it is what I do on my end to stay consistent with the version we use.

**Review by NickLaMuro — APPROVED** [review]
Looks like fine changes to me.  Left a suggestion, but I think it is pedantic, so probably not worth doing now.

File: developer_setup.md
Comment (kbrock) [reply developer_setup.md:77]: I too peg the version - I like nick's suggestion

Comment (Fryguy) [reply developer_setup.md:77]: yeah, I was debating that one myself.  I'm not sure how to peg the version in other systems.  @bdunne @agrare ?

Comment (Fryguy) [reply developer_setup.md]: I updated this one, but would like to update them all if possible.

Comment (NickLaMuro) [reply developer_setup.md]: That is why I was kinda of suggesting solving this another time.  I don't have familiarity with the other package managers like I do with `brew`, so I couldn't provide a solid answer, and I figured we should be consistent with this documentation.

Comment (agrare) [reply developer_setup.md]: Debian/Ubuntu it would be `apt install postgresql-10`

It depends on the version of your distro though, for example the latest stable version of debian `postgresql-11` is the oldest version available and I have `postgresql-12` and `postgresql-13` installed.  You would have to enable the prior version's apt source to get `apt install postgresql-10` to work.

Comment (Fryguy) [reply developer_setup.md]: Yeah this is why we avoided the version numbers 😆

Comment (agrare) [reply developer_setup.md]: Similar for Fedora, `dnf install postgresql-11` and 11 is the oldest version available on Fedora 33

Comment (Fryguy) [reply developer_setup.md]: I can just put 11 across the board in the docs.

Comment (agrare) [reply developer_setup.md]: > Yeah this is why we avoided the version numbers laughing

Haha yes, for devs it is probably better to have MIQ work with the latest version of PG (e.g. whatever `apt install postgresql` would bring in) even if it isn't "supported"

Comment (Fryguy) [reply developer_setup.md]: Thanks updated.  I also added a Min/Max to the initial table of versions, since installing 11 would be weird otherwise.

File: developer_setup.md
Comment (NickLaMuro) [inline developer_setup.md:101]: Still have `10` as a version number here.

File: developer_setup.md
Comment (NickLaMuro) [inline developer_setup.md:93]: Same here.

**Review by NickLaMuro — CHANGES_REQUESTED** [review]
Few spots where we are still using PG 10 for version numbers (since it seems like you went with pegging `11` in the docs now).

**Comment by Fryguy** [comment]
Thanks @NickLaMuro Updated.

**Review by NickLaMuro — APPROVED** [review]
Good from my perspective.

Wouldn't mind are resident "Debian distro expert", @agrare , confirm that the suggestions for `pg_dropcluster`, `pg_createcluster`, and `pg_ctlcluster` are version numbers for PG, and not some other `10` that was unrelated, but I couldn't see what else that might be off the top of my head.

**Comment by chessbyte** [comment]
Will merge after approval from @agrare and/or @bdunne regarding Linux distros.

**Comment by agrare** [comment]
> confirm that the suggestions for pg_dropcluster, pg_createcluster, and pg_ctlcluster are version numbers for PG, and not some other 10 that was unrelated

:+1: That's correct, `pg_dropcluster [--stop] cluster-version cluster-name` [[ref]](https://www.mankier.com/1/pg_dropcluster)

**Review by agrare — APPROVED** [review]

File: developer_setup.md
Comment (bdunne) [reply developer_setup.md]: @carbonin has suggested using https://www.postgresql.org/download/linux/redhat/ in the past

File: developer_setup.md
Comment (bdunne) [inline developer_setup.md:74]: This doesn't work for me on Fedora 33

Comment (agrare) [reply developer_setup.md:74]: Ah yeah seems like you can't install a class of versions like you can on deb, probably best to leave this as just `dnf -y install postgresql` then @bdunne ?

Comment (bdunne) [reply developer_setup.md:74]: Yes, `sudo dnf -y install postgresql-server`.  I assume it will be the same for `yum` as well.

Comment (Fryguy) [reply developer_setup.md]: Ok I put back the postgresql-server for dnf and yum

**Comment by Fryguy** [comment]
@bdunne this is ready to go

**Review by bdunne — APPROVED** [review]

**Pull request merged**
